js/setup.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+var userDialog = document.querySelector('.setup');
+userDialog.classList.remove('hidden');
+
+var WIZARD_NAME = ['Иван', 'Хуан', 'Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'];
+var WIZARD_SUBNAME = ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'];
+var WIZARD_COATCOLOR = ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'];
+var WIZARD_EYESCOLOR = ['black', 'red', 'blue', 'yellow', 'green'];
+
+
+var perem = function arrayRandElement(arr) {
+  var rand = Math.floor(Math.random() * arr.length);
+  var prm = arr[rand];
+  return prm;
```

Comment:
```suggestion
  return arr[rand];
```
Нету смысла заводить переменную для единичного использования на следующей же строке